src/h5web/visualizations/heatmap/HeatmapVis.module.css

```diff
@@ -19,37 +19,29 @@
 /* R3F's <Dom /> element*/
 .heatmap > div > div {
   transform: none !important;
-  pointer-events: none;
+  user-select: none; /* prevent text selection of tick labels when dragging */
```

Comment:
The axis can now be dragged to pan, so text selection is more problematic. I don't think people will need to select and copy tick labels, though.

Reply:
We will see in the future. Good to have the comment as a reminder.